tests/e2e/config/bootstrap.js

```diff
@@ -11,6 +11,7 @@ import {
 	enablePageDialogAccept,
 	setBrowserViewport,
 } from '@wordpress/e2e-test-utils';
+import { getQueryArg } from '@wordpress/url';
```

Comment:
FWIW this module is not explicitly listed as a dev dependency in `package.json`. I'd add it there instead of relying on it being there through some other way. In the browser it doesn't matter because it will use the `wp.url` anyway. So this is just needed for local development and tests.

Reply:
Added!